docker-compose.yml

```diff
@@ -1,46 +1,36 @@
 version: "3.5"
 services:
   node:
-    image: "node.hopr.public.dappnode.eth:1.0.27"
+    image: "node.hopr.public.dappnode.eth:1.0.19"
     build:
       context: .
       args:
-        UPSTREAM_VERSION: "2.0.7@sha256:cb63467df80840dc20b5f9f6d55977ede6e2902014dc67c86f3bbed3b69815ef"
+        UPSTREAM_VERSION: latest
     ports:
       - "9091:9091/tcp"
       - "9091:9091/udp"
       - "3001"
     volumes:
-      - "db:/app/hoprd-db"
+      - "data:/app/data"
     environment:
-      - DEBUG=hopr*
-      - HOPRD_ANNOUNCE=true
+      - RUST_LOG=info
       - HOPRD_NETWORK=dufour
-      - HOPRD_API=true
-      - HOPRD_API_HOST=0.0.0.0
-      - HOPRD_API_PORT=3001
-      - HOPRD_HEALTH_CHECK=true
-      - HOPRD_HEALTH_CHECK_HOST=0.0.0.0
-      - HOPRD_HEALTH_CHECK_PORT=8080
-      - HOPRD_API_TOKEN=!5qxc9Lp1BE7IFQ-nrtttU
-      - HOPRD_DISABLE_API_AUTHENTICATION=true
-      - HOPRD_IDENTITY=/app/hoprd-db/.hopr-identity
-      - HOPRD_DATA=/app/hoprd-db/data
-      - HOPRD_INIT=true
       - "HOPRD_HOST=127.0.0.1:9091"
       - HOPRD_SAFE_ADDRESS=0x
       - HOPRD_MODULE_ADDRESS=0x
-      - HOPRD_HEARTBEAT_THRESHOLD=60000
-      - HOPRD_HEARTBEAT_INTERVAL=20000
-      - HOPRD_PASSWORD="open-sesame-iTwnsPNg0hpagP+o6T0KOwiH9RQ0"
+      - HOPRD_HEARTBEAT_THRESHOLD=60
+      - HOPRD_HEARTBEAT_INTERVAL=20
+      - HOPRD_HEARTBEAT_VARIANCE=1
+      - HOPRD_PASSWORD=open-sesame-iTwnsPNg0hpagP+o6T0KOwiH9RQ0
       - "HOPRD_PROVIDER=https://provider-proxy.hoprnet.workers.dev/xdai_mainnet"
```

Comment:
HOPRD_PROVIDER change to `https://primary.gnosis-chain.rpc.hoprtech.net/` to avoid one proxy and access directly

Reply:
Will do.